src/QandaAtom.tsx

```diff
@@ -1,7 +1,445 @@
+                width="6px"
+                height="14px"
+                viewBox="0 0 6 14"
+            >
+                <path d="M4.6 12l-.4 1.4c-.7.2-1.9.6-3 .6-.7 0-1.2-.2-1.2-.9 0-.2 0-.3.1-.5l2-6.7H.7l.4-1.5 4.2-.6h.2L3 12h1.6zM4.3 2.8c-.9 0-1.4-.5-1.4-1.3C2.9.5 3.7 0 4.6 0 5.4 0 6 .5 6 1.3c0 1-.8 1.5-1.7 1.5z"></path>
+            </svg>
+        </span>
+        {' ' + credit}
+    </div>
+);
 
-export const QandaAtom = ({ id }: QandaAtomType): JSX.Element => (
-    <div data-atom-id={id}>QandaAtom</div>
+const Footer = ({
+    likeHandler,
+    dislikeHandler,
+}: {
+    likeHandler: () => void;
+    dislikeHandler: () => void;
+}) => {
+    const [showFeedback, setFeedbackVisibility] = useState(false);
+    return (
+        <footer className={footerStyling}>
+            <div hidden={showFeedback ? true : false}>
+                <div className={footerSnippet}>
+                    <div>Was this helpful?</div>
+                    <button
+                        className={buttonRound}
+                        value="like"
+                        aria-label="Yes"
+                        onClick={() => {
+                            likeHandler();
+                            setFeedbackVisibility(true);
+                        }}
+                    >
+                        <svg
+                            className={thumbIcon}
+                            width="40px"
+                            height="40px"
+                            viewBox="0 0 40 40"
+                        >
+                            <path
+                                fill="#FFF"
+                                d="M33.78 22.437l-4.228 13.98L27.93 37.5 5.062 34.14V15.503l7.8-1.517L24.354 2.5h1.624L28.9 5.426l-4.548 8.67h.107l10.477 1.31"
+                            ></path>
+                        </svg>
+                    </button>
+                    <button
+                        className={buttonRound}
+                        value="dislike"
+                        aria-label="No"
+                        onClick={() => {
+                            dislikeHandler();
+                            setFeedbackVisibility(true);
+                        }}
+                    >
+                        <svg
+                            className={thumbIcon + ' ' + dislikeThumb}
+                            width="40px"
+                            height="40px"
+                            viewBox="0 0 40 40"
+                        >
+                            <path
+                                fill="#FFF"
+                                d="M33.78 22.437l-4.228 13.98L27.93 37.5 5.062 34.14V15.503l7.8-1.517L24.354 2.5h1.624L28.9 5.426l-4.548 8.67h.107l10.477 1.31"
+                            ></path>
+                        </svg>
+                    </button>
+                </div>
+            </div>
+            <div
+                className={'feedback' + ' ' + footerFeedback}
```

Comment:
What's this 'feedback' class for?

Reply:
The reason I ask is that it's often an anti-pattern in DCR to see string classes like this

Reply:
I think that was left over from some tests I was doing, removed now